.github/workflows/pull-request-format.yml

```diff
@@ -0,0 +1,49 @@
+name: PR # Only a prefix to make it more readable in Github
+
+on:
+  push:
+    branches-ignore: [ master ]
```

Comment:
I would argue as this is designed for PRs entirely, this doesn't need to run on `push` at all.